### ISSUE TITLE
NVDA doesn't announce beep sound when changes to browse mode for document 

### ISSUE THREAD

**Issue by kolaps33** [nvaccess/nvda#15585]
Steps to reproduce:
1. Open the [chat example page](https://chataccessibility.z6.web.core.windows.net/iframe.html?args=&id=chat--chat-with-focusable-content&viewMode=story)

2. Run NVDA

3. Navigate on "before chat" button

4. Press TAB key (focus is on the message)

5. Press CTRL+ENTER (focus moves to the message text)

Actual behavior:
NVDA change to the browse mode and no sound indicated mode has changed to browse mode

Expected behavior:
NVDA announce with the sound changing the mode to browse mode

Comment:
The whole page has role application. This ensure user can navigate between chat messages with up/down arrow keys.
When focus is moved to message text content this element has role document. It allows user work with the text of the message. 
It works great. Only what we are missing is sound to inform user about mode has changed. 

NVDA logs, crash dumps and other attachments:
System configuration
NVDA installed/portable/running from source:
Installed

NVDA version:
Version: 2023.3beta4 (2023.3.0.29485)

Windows version:
Version 10.0.22621 Build 22621

Name and version of other software in use when reproducing the issue:
Chrome Version 117.0.5938.132 (Official Build) (64-bit)

Other information about your system:
Other questions
Does the issue still occur after restarting your computer?
yes

Have you tried any other versions of NVDA? If so, please report their behaviors.
no

If NVDA add-ons are disabled, is your problem still occurring?
yes

Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
didn't try

**Comment by michaelDCurran** [comment]
This is by design. As the entire page is role=application, then the subdocument in the chat is treated as its own entirely new document. NVDA never announces entering browse mode in a new document, such as opening a new browser window. Also as the  virtual caret is bound to the subdocument, this is  a new browse mode, not just toggling browse mode of the page itself.
